website/docs/r/vapp_network.html.markdown

```diff
@@ -15,14 +15,15 @@ Supported in provider *v2.1+*
 ## Example Usage
 
 ```hcl
+# IPv4 example
```

Comment:
Our website docs usually have such structure (each example has its own H2 markdown tag (2 hashes '##').

As a result Terraform registry docs show quick links to each of the examples 
<img width="974" alt="image" src="https://user-images.githubusercontent.com/15804230/225554322-297f3693-37ab-486a-8dbb-16f348862ec3.png">


```suggestion
```hcl
```